src/Physics/Plugins/ammoJSPlugin.ts

```diff
@@ -900,7 +900,16 @@ export class AmmoJSPlugin implements IPhysicsEnginePlugin {
 
         switch (impostor.type) {
             case PhysicsImpostor.SphereImpostor:
-                returnValue = new Ammo.btSphereShape(extendSize.x / 2);
+                // Is there a better way to compare floats number? With an epsylon or with a Math function

```

Comment:
You can use Scalar.WithinEpsilon

Reply:
tested this fix with my spinning ellipsoids, works perfectly.